lib/patches/net_patches.rb

```diff
@@ -1,16 +1,13 @@
 # frozen_string_literal: true
 
 if (defined?(Net) && defined?(Net::HTTP))
-
-  Net::HTTP.class_eval do
-    def request_with_mini_profiler(*args, &block)
-      request = args[0]
+  module NetHTTPWithMiniProfiler
+    def request(request, body = nil, &block)
```

Comment:
I kind of like the *args thing for safety in case more params are added down the line. Maybe keep that and we can go with the prepend.

Reply:
done. Kept both `request` and `*args` not to do `request = args[0]`